
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<meta content="Swiss Institute of Bioinformatics Literature Services" name="description"/>
<link href="https://sibils.org/vocabularies/" rel="canonical"/>
<link href="../collections/how_to_upload/" rel="prev"/>
<link href="../api/" rel="next"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.6.1, mkdocs-material-9.6.12" name="generator"/>
<title>Vocabularies - SIBiLS</title>
<link href="../assets/stylesheets/main.2afb09e1.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.06af60db.min.css" rel="stylesheet"/>
<link href="../assets/extra.css" rel="stylesheet"/>
<script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
<!-- Matomo -->
<script>
  var _paq = window._paq = window._paq || [];
  _paq.push(['trackPageView']);
  _paq.push(['enableLinkTracking']);
  (function() {
    var u="//matomo.text-analytics.ch/";
    _paq.push(['setTrackerUrl', u+'matomo.php']);
    _paq.push(['setSiteId', 10]);
    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
    g.type='text/javascript'; g.async=true; g.src=u+'matomo.js'; s.parentNode.insertBefore(g,s);
  })();
</script>
<!-- End Matomo Code -->
<script>"undefined"!=typeof __md_analytics&&__md_analytics()</script>
<meta content="website" property="og:type"/>
<meta content="Vocabularies - SIBiLS" property="og:title"/>
<meta content="Swiss Institute of Bioinformatics Literature Services" property="og:description"/>
<meta content="https://sibils.org/assets/images/social/vocabularies.png" property="og:image"/>
<meta content="image/png" property="og:image:type"/>
<meta content="1200" property="og:image:width"/>
<meta content="630" property="og:image:height"/>
<meta content="https://sibils.org/vocabularies/" property="og:url"/>
<meta content="summary_large_image" name="twitter:card"/>
<meta content="Vocabularies - SIBiLS" name="twitter:title"/>
<meta content="Swiss Institute of Bioinformatics Literature Services" name="twitter:description"/>
<meta content="https://sibils.org/assets/images/social/vocabularies.png" name="twitter:image"/>
</head>
<body data-md-color-accent="indigo" data-md-color-primary="indigo" data-md-color-scheme="sibils" dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#sibils-vocabularies-version-4256-2025">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="SIBiLS" class="md-header__button md-logo" data-md-component="logo" href=".." title="SIBiLS">
<img alt="logo" src="../assets/sib.png"/>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            SIBiLS
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Vocabularies
            
          </span>
</div>
</div>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<nav aria-label="Tabs" class="md-tabs" data-md-component="tabs">
<div class="md-grid">
<ul class="md-tabs__list">
<li class="md-tabs__item">
<a class="md-tabs__link" href="..">
        
  
  
    
  
  Introduction

      </a>
</li>
<li class="md-tabs__item">
<a class="md-tabs__link" href="../biodiversitypmc/">
        
  
  
    
  
  Biodiversity PMC

      </a>
</li>
<li class="md-tabs__item">
<a class="md-tabs__link" href="../biotxplorer/">
        
  
  
    
  
  BiotXplorer

      </a>
</li>
<li class="md-tabs__item">
<a class="md-tabs__link" href="../variomes/">
        
  
  
    
  
  Variomes

      </a>
</li>
<li class="md-tabs__item">
<a class="md-tabs__link" href="../synvar/">
        
  
  
    
  
  SynVar

      </a>
</li>
<li class="md-tabs__item">
<a class="md-tabs__link" href="../celltriage/">
        
  
  
    
  
  Celltriage

      </a>
</li>
<li class="md-tabs__item">
<a class="md-tabs__link" href="../sparql/">
        
  
  
    
  
  SPARQL

      </a>
</li>
<li class="md-tabs__item">
<a class="md-tabs__link" href="../collections/sibils_collections/">
          
  
  
    
  
  Collections

        </a>
</li>
<li class="md-tabs__item md-tabs__item--active">
<a class="md-tabs__link" href="./">
        
  
  
    
  
  Vocabularies

      </a>
</li>
<li class="md-tabs__item">
<a class="md-tabs__link" href="../api/">
          
  
  
    
  
  API

        </a>
</li>
<li class="md-tabs__item">
<a class="md-tabs__link" href="../sab/">
        
  
  
    
  
  SAB

      </a>
</li>
</ul>
</div>
</nav>
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden="">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary md-nav--lifted" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="SIBiLS" class="md-nav__button md-logo" data-md-component="logo" href=".." title="SIBiLS">
<img alt="logo" src="../assets/sib.png"/>
</a>
    SIBiLS
  </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
<span class="md-ellipsis">
    Introduction
    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../biodiversitypmc/">
<span class="md-ellipsis">
    Biodiversity PMC
    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../biotxplorer/">
<span class="md-ellipsis">
    BiotXplorer
    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../variomes/">
<span class="md-ellipsis">
    Variomes
    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../synvar/">
<span class="md-ellipsis">
    SynVar
    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../celltriage/">
<span class="md-ellipsis">
    Celltriage
    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../sparql/">
<span class="md-ellipsis">
    SPARQL
    
  </span>
</a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" id="__nav_8" type="checkbox"/>
<label class="md-nav__link" for="__nav_8" id="__nav_8_label" tabindex="0">
<span class="md-ellipsis">
    Collections
    
  </span>
<span class="md-nav__icon md-icon"></span>
</label>
<nav aria-expanded="false" aria-labelledby="__nav_8_label" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_8">
<span class="md-nav__icon md-icon"></span>
            Collections
          </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../collections/sibils_collections/">
<span class="md-ellipsis">
    SIBiLS collections
    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../collections/how_to_upload/">
<span class="md-ellipsis">
    upload publications in SIBiLS
    
  </span>
</a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--active">
<input class="md-nav__toggle md-toggle" id="__toc" type="checkbox"/>
<a class="md-nav__link md-nav__link--active" href="./">
<span class="md-ellipsis">
    Vocabularies
    
  </span>
</a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" id="__nav_10" type="checkbox"/>
<div class="md-nav__link md-nav__container">
<a class="md-nav__link" href="../api/">
<span class="md-ellipsis">
    API
    
  </span>
</a>
<label class="md-nav__link" for="__nav_10" id="__nav_10_label" tabindex="0">
<span class="md-nav__icon md-icon"></span>
</label>
</div>
<nav aria-expanded="false" aria-labelledby="__nav_10_label" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_10">
<span class="md-nav__icon md-icon"></span>
            API
          </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" id="__nav_10_2" type="checkbox"/>
<div class="md-nav__link md-nav__container">
<a class="md-nav__link" href="../api/biodiversitypmc/">
<span class="md-ellipsis">
    Biodiversity PMC
    
  </span>
</a>
<label class="md-nav__link" for="__nav_10_2" id="__nav_10_2_label" tabindex="0">
<span class="md-nav__icon md-icon"></span>
</label>
</div>
<nav aria-expanded="false" aria-labelledby="__nav_10_2_label" class="md-nav" data-md-level="2">
<label class="md-nav__title" for="__nav_10_2">
<span class="md-nav__icon md-icon"></span>
            Biodiversity PMC
          </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../api/biodiversitypmc/fetch/">
<span class="md-ellipsis">
    Fetch
    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/biodiversitypmc/search/">
<span class="md-ellipsis">
    Search
    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/biodiversitypmc/qa/">
<span class="md-ellipsis">
    QA
    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/biodiversitypmc/pmid2doi/">
<span class="md-ellipsis">
    PMID2DOI
    
  </span>
</a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/celltriage/">
<span class="md-ellipsis">
    Celltriage
    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/variomes/">
<span class="md-ellipsis">
    Variomes
    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/synvar/">
<span class="md-ellipsis">
    SynVar
    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../api/sparql/">
<span class="md-ellipsis">
    SPARQL
    
  </span>
</a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../sab/">
<span class="md-ellipsis">
    SAB
    
  </span>
</a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" hidden="">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<h1 id="sibils-vocabularies-version-4256-2025">SIBiLS Vocabularies - version 4.2.5.6 (2025)</h1>
<table>
<thead>
<tr>
<th>Name</th>
<th>Code</th>
<th>Type</th>
<th>Update / Version</th>
<th>Description</th>
<th>Link</th>
<th>Concept_source</th>
</tr>
</thead>
<tbody>
<tr>
<td>Accession numbers</td>
<td>identifiers</td>
<td>accession_numbers</td>
<td>v01-2025</td>
<td>A list of patterns based on the registry of identifiers.org and enriched manually.</td>
<td><a href="https://identifiers.org">Identifiers.org</a></td>
<td>accessionnumbers</td>
</tr>
<tr>
<td>Affiliations</td>
<td>ror</td>
<td>affiliation</td>
<td>v2025-01-23</td>
<td>A list of international research institutes.</td>
<td><a href="https://ror.org">ROR</a></td>
<td>ror</td>
</tr>
<tr>
<td>ATC</td>
<td>atc</td>
<td>drug</td>
<td>v2024AB</td>
<td>This ontology is a representation of the ATC classification used for the classification of drugs and provided by WHO (World Health Organization). Based on the ATC code of the main ingredient of a product, this medical product can be classify in the ATC system.</td>
<td><a href="https://bioportal.bioontology.org/ontologies/ATC">ATC</a></td>
<td>atc</td>
</tr>
<tr>
<td>Agrovoc</td>
<td>agrovoc</td>
<td>agriculture</td>
<td>v2024-06-04</td>
<td>AGROVOC covers all areas of interest to FAO, such as food, nutrition, agriculture, forestry, fisheries, names of animals and plants, environment, biological notions, techniques of plant cultivation, etc. The thesaurus is hierarchically organized under 25 top concepts.</td>
<td><a href="https://www.fao.org/agrovoc/releases">Agrovoc</a></td>
<td>agrovoc</td>
</tr>
<tr>
<td>Cell Ontology</td>
<td>celltype</td>
<td>cell type</td>
<td>2025-01-07</td>
<td>The Cell Ontology is a structured controlled vocabulary for cell types in animals.</td>
<td><a href="http://obofoundry.org/ontology/cl.html">Cell Ontology</a></td>
<td>cellontology</td>
</tr>
<tr>
<td>Cellosaurus</td>
<td>cellosaurus</td>
<td>cell line</td>
<td>51.0</td>
<td>The Cellosaurus is a knowledge resource on cell lines. It attempts to describe all cell lines used in biomedical research. Its scope includes: Immortalized cell lines; Naturally immortal cell lines (example: stem cell lines); Finite life cell lines when those are distributed and used widely; Vertebrate cell line with an emphasis on human, mouse and rat cell lines; Invertebrate (insects and ticks) cell lines. Its scope does not include: Primary cell lines (with the exception of the finite life cell lines described above); Plant cell lines</td>
<td><a href="https://www.cellosaurus.org">Cellosaurus</a></td>
<td>cellosaurus</td>
</tr>
<tr>
<td>CheBI (Chemical Entities of Biological Interest)</td>
<td>chebi</td>
<td>chemical</td>
<td>v239</td>
<td>Chemical Entities of Biological Interest (ChEBI) is a freely available dictionary of molecular entities focused on ‘small’ chemical compounds. The term ‘molecular entity’ refers to any constitutionally or isotopically distinct atom, molecule, ion, ion pair, radical, radical ion, complex, conformer, etc., identifiable as a separately distinguishable entity. The molecular entities in question are either products of nature or synthetic products used to intervene in the processes of living organisms. ChEBI incorporates an ontological classification, whereby the relationships between molecular entities or classes of entities and their parents and/or children are specified.</td>
<td><a href="https://www.ebi.ac.uk/chebi/downloadsForward.do">ChEBI</a></td>
<td>chebi</td>
</tr>
<tr>
<td>COVoc</td>
<td></td>
<td></td>
<td>2022</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more. Several axis are detailed below and accessible via SIBiLS.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
<td></td>
</tr>
<tr>
<td>COVoc Biomedical</td>
<td>cov</td>
<td>biomedical_vocab</td>
<td>v1.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
<td>covocbiomed</td>
</tr>
<tr>
<td>COVoc Cell lines</td>
<td>cov</td>
<td>cell_lines</td>
<td>v1.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
<td>covoccelllines</td>
</tr>
<tr>
<td>COVoc Chemicals</td>
<td>cov</td>
<td>chemical</td>
<td>v2.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
<td>covocchemicals</td>
</tr>
<tr>
<td>COVoc Clinical Trials</td>
<td>cov</td>
<td>clinical_trial</td>
<td>v1.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
<td>covocclinicaltrials</td>
</tr>
<tr>
<td>COVoc Conceptual Entities</td>
<td>cov</td>
<td>conceptual_entity</td>
<td>v1.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
<td>covocconceptualentities</td>
</tr>
<tr>
<td>COVoc Disease and Syndrome</td>
<td>cov</td>
<td>disease_syndrome</td>
<td>v1.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
<td>covocdiseaseandsyndrom</td>
</tr>
<tr>
<td>COVoc Geographic Location</td>
<td>cov</td>
<td>geographic_location</td>
<td>v1.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
<td>covocgeographicloc</td>
</tr>
<tr>
<td>COVoc Organism</td>
<td>cov</td>
<td>species</td>
<td>v2.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
<td>covocorganism</td>
</tr>
<tr>
<td>COVoc Protein and Genome</td>
<td>cov</td>
<td>protein_genome</td>
<td>v2.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
<td>covocproteinsgenomes</td>
</tr>
<tr>
<td>Detection methods</td>
<td>vdm</td>
<td>method</td>
<td>v2022</td>
<td>List of methods used to detect or study viruses in a corpus of scientific texts.</td>
<td>[Private]</td>
<td>detectionmethods</td>
</tr>
<tr>
<td>Disprot</td>
<td>disprot</td>
<td>disprot_type1</td>
<td>v1.0</td>
<td>xxx</td>
<td><a href="xxx">xxx</a></td>
<td>disprot_type1</td>
</tr>
<tr>
<td>Disprot</td>
<td>disprot</td>
<td>disprot_type2</td>
<td>v1.0</td>
<td>xxx</td>
<td><a href="xxx">xxx</a></td>
<td>disprot_type2</td>
</tr>
<tr>
<td>Disprot</td>
<td>disprot</td>
<td>disprot_type3</td>
<td>v1.0</td>
<td>xxx</td>
<td><a href="xxx">xxx</a></td>
<td>disprot_type3</td>
</tr>
<tr>
<td>Disprot</td>
<td>disprot</td>
<td>disprot_type4</td>
<td>v1.0</td>
<td>xxx</td>
<td><a href="xxx">xxx</a></td>
<td>disprot_type4</td>
</tr>
<tr>
<td>Drugbank</td>
<td>drugbank</td>
<td>drug</td>
<td>v5.1.13</td>
<td>The Drugbank database is a freely accessible resource which includes more than 13,000 records (version 5.1.4, released 2019-07-20). It contains information on drugs and drug targets, synonyms and product names.</td>
<td><a href="https://www.drugbank.ca/releases/latest">Drugbank</a></td>
<td>drugbank</td>
</tr>
<tr>
<td>ECO (Evidence and Conclusion Ontology)</td>
<td>eco</td>
<td>evidence</td>
<td>v2024-07-19</td>
<td>The Evidence &amp; Conclusion Ontology (ECO) describes types of scientific evidence within the biological research domain that arise from laboratory experiments, computational methods, literature curation, or other means.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/eco">ECO</a></td>
<td>eco</td>
</tr>
<tr>
<td>ENVO (Environment Ontology)</td>
<td>envo</td>
<td>environment</td>
<td>v2024-07-01</td>
<td>ENVO is an expressive, community ontology which helps humans, machines, and semantic web applications understand environmental entities of all kinds, from microscopic to intergalactic scales.</td>
<td><a href="https://obofoundry.org/ontology/envo.html">ENVO</a></td>
<td>envo</td>
</tr>
<tr>
<td>Flopo</td>
<td>flopo</td>
<td>plant</td>
<td>2025-01-28</td>
<td>The flora phenotype ontology (FLOPO): tool for integrating morphological traits and phenotypes of vascular plants..</td>
<td><a href="https://obofoundry.org/ontology/flopo.html">Flopo</a></td>
<td>flopo</td>
</tr>
<tr>
<td>GO (Gene Ontology)</td>
<td>go</td>
<td>biological_process</td>
<td>v2024-11-03</td>
<td>The Gene Ontology (GO) knowledgebase is the world's largest source of information on the functions of genes. This knowledge is both human-readable and machine-readable, and is a foundation for computational analysis of large-scale molecular biology and genetics experiments in biomedical research. Specific type: Biological process.</td>
<td><a href="http://geneontology.org/">Gene Ontology</a></td>
<td>go_bp</td>
</tr>
<tr>
<td>GO (Gene Ontology)</td>
<td>go</td>
<td>cellular_component</td>
<td>v2024-11-03</td>
<td>The Gene Ontology (GO) knowledgebase is the world's largest source of information on the functions of genes. This knowledge is both human-readable and machine-readable, and is a foundation for computational analysis of large-scale molecular biology and genetics experiments in biomedical research. Specific type: Cellular component.</td>
<td><a href="http://geneontology.org/">Gene Ontology</a></td>
<td>go_cc</td>
</tr>
<tr>
<td>GO (Gene Ontology)</td>
<td>go</td>
<td>molecular_function</td>
<td>v2024-11-03</td>
<td>The Gene Ontology (GO) knowledgebase is the world's largest source of information on the functions of genes. This knowledge is both human-readable and machine-readable, and is a foundation for computational analysis of large-scale molecular biology and genetics experiments in biomedical research. Specific type: Molecular function.</td>
<td><a href="http://geneontology.org/">Gene Ontology</a></td>
<td>go_mf</td>
</tr>
<tr>
<td>ICD-O-3</td>
<td>icdo3</td>
<td>disease</td>
<td>unknown</td>
<td>The International Classification of Diseases Oncology (ICD-O) is a biomedical ontology for logical representation of the terms and relations related to the International Classification of Diseases (ICD).</td>
<td><a href="http://www.iacr.com.fr/index.php?option=com_content&amp;view=category&amp;layout=blog&amp;id=100&amp;Itemid=577">ICD-O-3</a></td>
<td>icdo3</td>
</tr>
<tr>
<td>ICTV (International Committee on Taxonomy of Viruses)</td>
<td>ictv</td>
<td>virus</td>
<td>MSL39v4</td>
<td>The task of the International Committee on Taxonomy of Viruses (ICTV) is to develop a single, universal taxonomic scheme for all the viruses infecting animals (vertebrates, invertebrates and protozoa), plants (higher plants and algae), fungi, bacteria and archaea.</td>
<td><a href="https://ictv.global/taxonomy">ICTV</a></td>
<td>ictv</td>
</tr>
<tr>
<td>License</td>
<td>license</td>
<td>license</td>
<td>v3.26</td>
<td>The SPDX License List itself is a list of commonly found licenses and exceptions used in free and open or collaborative software, data, hardware, or documentation.</td>
<td><a href="https://github.com/spdx/license-list-data/tree/main">Licenses (SPDX)</a></td>
<td>license</td>
</tr>
<tr>
<td>LOTUS</td>
<td>lotus</td>
<td>chemical</td>
<td>v10</td>
<td>LOTUS, one of the biggest and best annotated resources for Natural Products occurrences available free of charge and without any restriction.</td>
<td><a href="https://lotus.nprod.net">LOTUS</a></td>
<td>lotus</td>
</tr>
<tr>
<td>MDD (Mammal Diversity Database)</td>
<td>mdd</td>
<td>species</td>
<td>v1.13</td>
<td>Mammal species list with synonyms. An initial taxonomy of accepted names + synonyms based on v1.7 of the Mammal Diversity Database (MDD)</td>
<td><a href="https://www.mammaldiversity.org/index.html">MDD</a></td>
<td>mdd</td>
</tr>
<tr>
<td>MeSH</td>
<td>mesh</td>
<td>mesh</td>
<td>v2025</td>
<td>The Medical Subject Headings (MeSH), provided by the U.S. National Library of Medicine (NLM), is a controlled vocabulary thesaurus used for indexing articles for PubMed. In comparison with specialzed ontologies like the NCIt, MeSH is less granular and easily identified by Natural Language Processing thanks to synonyms.</td>
<td><a href="https://www.nlm.nih.gov/databases/download/mesh.html">MeSH</a></td>
<td>mesh</td>
</tr>
<tr>
<td>NCBI Taxonomy Full</td>
<td>ncbi</td>
<td>species</td>
<td>v.2025-02-11</td>
<td>The NCBI Taxonomy Database is a curated classification and nomenclature for all of the organisms in the public sequence databases.</td>
<td><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi">NCBI Taxonomy</a></td>
<td>ncbitaxon_full</td>
</tr>
<tr>
<td>NCBI Taxonomy Models</td>
<td>ncbi</td>
<td>species</td>
<td>v.2025-02-11</td>
<td>This terminology was created by the team and includes many species of clinical interest and related code from NCBI taxonomy collection.</td>
<td><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi">NCBI Taxonomy</a></td>
<td>ncbitaxon_models</td>
</tr>
<tr>
<td>NCI Thesaurus</td>
<td>ncit</td>
<td>disease</td>
<td>v24.12e</td>
<td>The NCI Thesaurus (NCIt) is used for disease mapping. It covers clinical care, translational and basic research, public information and administrative activities. Provided by the National Cancer Institute, this terminology is a standard for biomedical coding and reference, used both by public and private scientific partners worldwide. This NCI's reference terminology contains the NCI_CUI, the semantic type, a prefered term, some NCI and MeSH synonyms.</td>
<td><a href="https://ncithesaurus.nci.nih.gov/ncitbrowser/">NCIt</a></td>
<td>ncit</td>
</tr>
<tr>
<td>neXtProt</td>
<td>nextprot</td>
<td>gene</td>
<td>v2023-09-12</td>
<td>Developed by the SIB (Swiss Institute of Bioinformatics) in 2008, the neXtProt human protein knowledgebase is a comprehensive human-centric discovery platform. More than 20,000 proteins were manually annotated and still updated. The provides to researchers a high-quality synonym for both protein and gene names.</td>
<td><a href="https://www.nextprot.org/">neXtProt</a></td>
<td>nextprot</td>
</tr>
<tr>
<td>OTT (Open Tree of Life)</td>
<td>ott</td>
<td>species</td>
<td>v3.7.2</td>
<td>Open Tree of Life aims to construct a comprehensive, dynamic and digitally-available tree of life by synthesizing published phylogenetic trees along with taxonomic data. https://doi.org/10.5281/zenodo.3937750</td>
<td><a href="https://tree.opentreeoflife.org/about/taxonomy-version/ott3.3">OTT</a></td>
<td>ott</td>
</tr>
<tr>
<td>PO (Plant Ontology)</td>
<td>po</td>
<td>plant</td>
<td>04-18-2024</td>
<td>The Plant Ontology is a structured vocabulary and database resource that links plant anatomy, morphology and growth and development to plant genomics data.</td>
<td><a href="https://obofoundry.org/ontology/po">PO</a></td>
<td>po</td>
</tr>
<tr>
<td>PPI-PTM</td>
<td>ppiptm</td>
<td>ppi-ptm</td>
<td>unknown</td>
<td></td>
<td>[Manual]</td>
<td>ppiptm</td>
</tr>
<tr>
<td>PSI-MI</td>
<td>psimi</td>
<td>molecular_interaction</td>
<td>v1.2</td>
<td>A structured controlled vocabulary for the annotation of experiments concerned with protein-protein interactions.</td>
<td><a href="https://www.ebi.ac.uk/ols4/ontologies/mi">psi-mi</a></td>
<td>psimi</td>
</tr>
<tr>
<td>Pubchem (Mesh subset)</td>
<td>pubchemmesh</td>
<td>chemical</td>
<td>unknown</td>
<td>Pubchem database, only coumpounds with mesh terms associated. Reconstitution pipeline from FTP, Extras files.</td>
<td><a href="https://ftp.ncbi.nlm.nih.gov/pubchem/Compound/">pubchem</a></td>
<td>pubchemmesh</td>
</tr>
<tr>
<td>ROBI (Relation Ontology - Biotic Interactions)</td>
<td>robiext</td>
<td>biotic_interaction</td>
<td>v2025</td>
<td>RO is a collection of relations intended primarily for standardization across ontologies in the OBO Foundry and wider OBO library. It incorporates ROCore upper-level relations such as part of as well as biology-specific relationship types such as develops from. We extracted a subset of terms of interest for biotic interactions.</td>
<td><a href="https://obofoundry.org/ontology/ro.html">RO</a></td>
<td>robiext</td>
</tr>
<tr>
<td>UniProtKB/SwissProt</td>
<td>uniprot</td>
<td>gene</td>
<td>v11</td>
<td>The UniProt Knowledgebase (UniProtKB) is the central hub for the collection of functional information on proteins, with accurate, consistent and rich annotation. We used a large fraction of the reviewed subset UinProtKB/Swiss-Prot which includes records with information extracted from literature and curator-evaluated computational analysis.</td>
<td><a href="https://www.uniprot.org/downloads#uniprotkblink">UniProtKB/SwissProt</a></td>
<td>uniprot_swissprot</td>
</tr>
</tbody>
</table>
<p>Correspondance between definition, json_field and annotation_field:</p>
<table>
<thead>
<tr>
<th>definition</th>
<th>json_field</th>
<th>annotation_field</th>
</tr>
</thead>
<tbody>
<tr>
<td>terminology name</td>
<td>terminology</td>
<td>concept_source</td>
</tr>
<tr>
<td>terminology version</td>
<td>version</td>
<td>version</td>
</tr>
<tr>
<td>release date</td>
<td>date</td>
<td>null</td>
</tr>
<tr>
<td>terminology type</td>
<td>type</td>
<td>type</td>
</tr>
</tbody>
</table>
<p>Correspondance between type and concept_source </p>
<table>
<thead>
<tr>
<th>type</th>
<th>concept_source</th>
</tr>
</thead>
<tbody>
<tr>
<td>accession_numbers</td>
<td>accessionnumbers</td>
</tr>
<tr>
<td>affiliation</td>
<td>ror</td>
</tr>
<tr>
<td>agriculture</td>
<td>agrovoc</td>
</tr>
<tr>
<td>biological_process</td>
<td>go_bp</td>
</tr>
<tr>
<td>biomedical_vocab</td>
<td>covocbiomed</td>
</tr>
<tr>
<td>biotic_interaction</td>
<td>robiext</td>
</tr>
<tr>
<td>cell_lines</td>
<td>covoccelllines</td>
</tr>
<tr>
<td>cell line</td>
<td>cellosaurus</td>
</tr>
<tr>
<td>cell type</td>
<td>cellontology</td>
</tr>
<tr>
<td>cellular_component</td>
<td>go_cc</td>
</tr>
<tr>
<td>chemical</td>
<td>chebi, covocchemicals, lotus, pubchemmesh</td>
</tr>
<tr>
<td>clinical_trial</td>
<td>covocclinicaltrials</td>
</tr>
<tr>
<td>conceptual_entity</td>
<td>covocconceptualentities</td>
</tr>
<tr>
<td>disease</td>
<td>icdo3, ncit</td>
</tr>
<tr>
<td>disease_syndrome</td>
<td>covocdiseaseandsyndrom</td>
</tr>
<tr>
<td>disprot_type1</td>
<td>disprot_type1</td>
</tr>
<tr>
<td>disprot_type2</td>
<td>disprot_type2</td>
</tr>
<tr>
<td>disprot_type3</td>
<td>disprot_type3</td>
</tr>
<tr>
<td>disprot_type4</td>
<td>disprot_type4</td>
</tr>
<tr>
<td>drug</td>
<td>atc, drugbank</td>
</tr>
<tr>
<td>evidence</td>
<td>eco</td>
</tr>
<tr>
<td>environment</td>
<td>envo</td>
</tr>
<tr>
<td>geographic_location</td>
<td>covocgeographicloc</td>
</tr>
<tr>
<td>gene</td>
<td>nextprot, uniprot_swissprot</td>
</tr>
<tr>
<td>license</td>
<td>license</td>
</tr>
<tr>
<td>mesh</td>
<td>mesh</td>
</tr>
<tr>
<td>method</td>
<td>detectionmethods</td>
</tr>
<tr>
<td>molecular_function</td>
<td>go_mf</td>
</tr>
<tr>
<td>molecular_interaction</td>
<td>psi-mi</td>
</tr>
<tr>
<td>plant</td>
<td>flopo, po</td>
</tr>
<tr>
<td>ppi-ptm</td>
<td>ppi-ptm</td>
</tr>
<tr>
<td>protein_genome</td>
<td>covocproteinsgenomes</td>
</tr>
<tr>
<td>species</td>
<td>mdd, ott, covocorganism, ncbitaxon_full, ncbitaxon_models</td>
</tr>
<tr>
<td>virus</td>
<td>ictv</td>
</tr>
</tbody>
</table>
</article>
</div>
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
<div class="md-copyright__highlight">
        Tagged entities are licensed under a <a href="http://creativecommons.org/licenses/by/3.0/ch/">Creative Commons Attribution 3.0 Switzerland License</a>.
    </div>
<div>
        Questions specifically relating to these services may be sent to <a href="mailto:julien.gobeill@hesge.ch">julien.gobeill@hesge.ch</a>.
    </div>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.indexes", "content.code.copy"], "search": "../assets/javascripts/workers/search.f8cc74c7.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
<script src="../assets/javascripts/bundle.c8b220af.min.js"></script>
<script>document$.subscribe(() => {
            window.update_swagger_ui_iframe_height = function (id) {
                var iFrameID = document.getElementById(id);
                if (iFrameID) {
                    full_height = (iFrameID.contentWindow.document.body.scrollHeight + 80) + "px";
                    iFrameID.height = full_height;
                    iFrameID.style.height = full_height;
                }
            }
        
            let iframe_id_list = []
            var iframes = document.getElementsByClassName("swagger-ui-iframe");
            for (var i = 0; i < iframes.length; i++) { 
                iframe_id_list.push(iframes[i].getAttribute("id"))
            }
        
            let ticking = true;
            
            document.addEventListener('scroll', function(e) {
                if (!ticking) {
                    window.requestAnimationFrame(()=> {
                        let half_vh = window.innerHeight/2;
                        for(var i = 0; i < iframe_id_list.length; i++) {
                            let element = document.getElementById(iframe_id_list[i])
                            if(element==null){
                                return
                            }
                            let diff = element.getBoundingClientRect().top
                            if(element.contentWindow.update_top_val){
                                element.contentWindow.update_top_val(half_vh - diff)
                            }
                        }
                        ticking = false;
                    });
                    ticking = true;
                }
            });
        
            const dark_scheme_name = "slate"
            
            window.scheme = document.body.getAttribute("data-md-color-scheme")
            const options = {
                attributeFilter: ['data-md-color-scheme'],
            };
            function color_scheme_callback(mutations) {
                for (let mutation of mutations) {
                    if (mutation.attributeName === "data-md-color-scheme") {
                        scheme = document.body.getAttribute("data-md-color-scheme")
                        var iframe_list = document.getElementsByClassName("swagger-ui-iframe")
                        for(var i = 0; i < iframe_list.length; i++) {
                            var ele = iframe_list.item(i);
                            if (ele) {
                                if (scheme === dark_scheme_name) {
                                    ele.contentWindow.enable_dark_mode();
                                } else {
                                    ele.contentWindow.disable_dark_mode();
                                }
                            }
                        }
                    }
                }
            }
            observer = new MutationObserver(color_scheme_callback);
            observer.observe(document.body, options);
            })</script></body>
</html>