
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Swiss Institute of Bioinformatics Literature Services">
      
      
      
        <link rel="canonical" href="https://sibils.github.io/doc/vocabularies/">
      
      
        <link rel="prev" href="../..">
      
      
        <link rel="next" href="../api/fetch_medline/">
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.4.2, mkdocs-material-9.1.6">
    
    
      
        <title>Vocabularies - SIBiLS</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.ded33207.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.a0c5b2b5.min.css">
      
      

    
    
    
      
    
    
      <link rel="stylesheet" href="../../assets/extra.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
      
        <meta  property="og:type"  content="website" >
      
        <meta  property="og:title"  content="Vocabularies - SIBiLS" >
      
        <meta  property="og:description"  content="Swiss Institute of Bioinformatics Literature Services" >
      
        <meta  property="og:image"  content="https://sibils.github.io/assets/images/social/doc/vocabularies.png" >
      
        <meta  property="og:image:type"  content="image/png" >
      
        <meta  property="og:image:width"  content="1200" >
      
        <meta  property="og:image:height"  content="630" >
      
        <meta  property="og:url"  content="https://sibils.github.io/doc/vocabularies/" >
      
        <meta  name="twitter:card"  content="summary_large_image" >
      
        <meta  name="twitter:title"  content="Vocabularies - SIBiLS" >
      
        <meta  name="twitter:description"  content="Swiss Institute of Bioinformatics Literature Services" >
      
        <meta  name="twitter:image"  content="https://sibils.github.io/assets/images/social/doc/vocabularies.png" >
      
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="sibils" data-md-color-primary="" data-md-color-accent="">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#sibils-vocabularies-version-31-2023" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="SIBiLS" class="md-header__button md-logo" aria-label="SIBiLS" data-md-component="logo">
      
  <img src="../../assets/sib.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            SIBiLS
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Vocabularies
            
          </span>
        </div>
      </div>
    </div>
    
    
    
    
      <div class="md-header__source">
        <a href="https://github.com/sibils/sibils.github.io/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  


  <li class="md-tabs__item">
    <a href="../.." class="md-tabs__link">
      Quick Start
    </a>
  </li>

      
        
  
  
    
  


  <li class="md-tabs__item">
    <a href="./" class="md-tabs__link md-tabs__link--active">
      Vocabularies
    </a>
  </li>

      
        
  
  


  
  
  
    <li class="md-tabs__item">
      <a href="../api/fetch_medline/" class="md-tabs__link">
        API
      </a>
    </li>
  

      
        
  
  


  <li class="md-tabs__item">
    <a href="../../search/" class="md-tabs__link">
      SIBiLS Search
    </a>
  </li>

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="SIBiLS" class="md-nav__button md-logo" aria-label="SIBiLS" data-md-component="logo">
      
  <img src="../../assets/sib.png" alt="logo">

    </a>
    SIBiLS
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/sibils/sibils.github.io/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        Quick Start
      </a>
    </li>
  

    
      
      
      

  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          Vocabularies
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        Vocabularies
      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#mapped-vocabularies" class="md-nav__link">
    Mapped vocabularies
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
      
      
      <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
      
      
        
          
        
          
        
          
        
          
        
          
        
      
      
        <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
          API
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
        <label class="md-nav__title" for="__nav_3">
          <span class="md-nav__icon md-icon"></span>
          API
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../api/fetch_medline/" class="md-nav__link">
        fetch in MEDLINE
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../api/fetch_pmc/" class="md-nav__link">
        fetch in PubMed Central
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../api/search_medline/" class="md-nav__link">
        search in MEDLINE
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../api/search_pmc/" class="md-nav__link">
        search in PubMed Central
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../api/sparql/" class="md-nav__link">
        SPARQL
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../../search/" class="md-nav__link">
        SIBiLS Search
      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#mapped-vocabularies" class="md-nav__link">
    Mapped vocabularies
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<h1 id="sibils-vocabularies-version-31-2023">SIBiLS Vocabularies - version 3.1 (2023)</h1>
<h2 id="mapped-vocabularies">Mapped vocabularies</h2>
<p>"An ontology is a formal representation of a body of knowledges within a given domain. Ontologies usually consist of a set of classes (or terms or concepts) with relations that operate between them."</p>
<p>These are the various terminologies used to create annotations.</p>
<table>
<thead>
<tr>
<th>Name</th>
<th>Code</th>
<th>Type</th>
<th>Update / Version</th>
<th>Description</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr>
<td>Affiliations</td>
<td>aff</td>
<td>affiliation</td>
<td>2022</td>
<td>A list of international research institutes.</td>
<td>[Manual]</td>
</tr>
<tr>
<td>ATC</td>
<td>atc</td>
<td>drug</td>
<td>2022</td>
<td>This ontology is a representation of the ATC classification used for the classification of drugs and provided by WHO (World Health Organization). Based on the ATC code of the main ingredient of a product, this medical product can be classify in the ATC system.</td>
<td><a href="https://bioportal.bioontology.org/ontologies/ATC">ATC</a></td>
</tr>
<tr>
<td>Cellosaurus</td>
<td>cell</td>
<td>cell line</td>
<td>v43.0</td>
<td>xxx</td>
<td><a href="https://www.cellosaurus.org">Cellosaurus</a></td>
</tr>
<tr>
<td>CheBI (Chemical Entities of Biological Interest)</td>
<td>chebi</td>
<td>chemical</td>
<td>v220</td>
<td>Chemical Entities of Biological Interest (ChEBI) is a freely available dictionary of molecular entities focused on ‘small’ chemical compounds. The term ‘molecular entity’ refers to any constitutionally or isotopically distinct atom, molecule, ion, ion pair, radical, radical ion, complex, conformer, etc., identifiable as a separately distinguishable entity. The molecular entities in question are either products of nature or synthetic products used to intervene in the processes of living organisms. ChEBI incorporates an ontological classification, whereby the relationships between molecular entities or classes of entities and their parents and/or children are specified.</td>
<td><a href="https://www.ebi.ac.uk/chebi/downloadsForward.do">ChEBI</a></td>
</tr>
<tr>
<td>COVoc</td>
<td>cov</td>
<td>COVoc</td>
<td>2022</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
</tr>
<tr>
<td>COVoc Biomedical</td>
<td>cov</td>
<td>biomedical_vocab</td>
<td>v1.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
</tr>
<tr>
<td>COVoc Cell lines</td>
<td>cov</td>
<td>cell_lines</td>
<td>v1.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
</tr>
<tr>
<td>COVoc Chemicals</td>
<td>cov</td>
<td>chemical</td>
<td>v2.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
</tr>
<tr>
<td>COVoc Clinical Trials</td>
<td>cov</td>
<td>clinical_trial</td>
<td>v1.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
</tr>
<tr>
<td>COVoc Conceptual Entities</td>
<td>cov</td>
<td>conceptual_entity</td>
<td>v1.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
</tr>
<tr>
<td>COVoc Disease and Syndrome</td>
<td>cov</td>
<td>disease_syndrome</td>
<td>v1.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
</tr>
<tr>
<td>COVoc Geographic Location</td>
<td>cov</td>
<td>geographic_location</td>
<td>v1.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
</tr>
<tr>
<td>COVoc Organism</td>
<td>cov</td>
<td>species</td>
<td>v2.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
</tr>
<tr>
<td>COVoc Protein and Genome</td>
<td>cov</td>
<td>protein_genome</td>
<td>v2.0</td>
<td>The COVID-19 Vocabulary (COVoc) is an ontology containing terms related to the research of the COVID-19 pandemic. This includes host organisms, pathogenicity, gene and gene products, barrier gestures, treatments and more.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/COVOC">COVoc</a></td>
</tr>
<tr>
<td>Detection methods</td>
<td>vdm</td>
<td>method</td>
<td>2022</td>
<td>List of methods used to detect or study viruses in a corpus of scientific texts.</td>
<td>[Private]</td>
</tr>
<tr>
<td>Disprot</td>
<td>disprot</td>
<td>disprot_type1</td>
<td>v1.0</td>
<td>xxx</td>
<td><a href="xxx">xxx</a></td>
</tr>
<tr>
<td>Disprot</td>
<td>disprot</td>
<td>disprot_type2</td>
<td>v1.0</td>
<td>xxx</td>
<td><a href="xxx">xxx</a></td>
</tr>
<tr>
<td>Disprot</td>
<td>disprot</td>
<td>disprot_type3</td>
<td>v1.0</td>
<td>xxx</td>
<td><a href="xxx">xxx</a></td>
</tr>
<tr>
<td>Disprot</td>
<td>disprot</td>
<td>disprot_type4</td>
<td>v1.0</td>
<td>xxx</td>
<td><a href="xxx">xxx</a></td>
</tr>
<tr>
<td>Drugbank</td>
<td>drugbank</td>
<td>drug</td>
<td>v5.1.5</td>
<td>The Drugbank database is a freely accessible resource which includes more than 13,000 records (version 5.1.4, released 2019-07-20). It contains information on drugs and drug targets, synonyms and product names.</td>
<td><a href="https://www.drugbank.ca/releases/latest">Drugbank</a></td>
</tr>
<tr>
<td>ECO (Evidence and Conclusion Ontology)</td>
<td>eco</td>
<td>evidence</td>
<td>2022/08/05</td>
<td>The Evidence &amp; Conclusion Ontology (ECO) describes types of scientific evidence within the biological research domain that arise from laboratory experiments, computational methods, literature curation, or other means.</td>
<td><a href="https://www.ebi.ac.uk/ols/ontologies/eco">ECO</a></td>
</tr>
<tr>
<td>ENVO</td>
<td>envo</td>
<td>environment</td>
<td>2021/05/14</td>
<td>ENVO is an expressive, community ontology which helps humans, machines, and semantic web applications understand environmental entities of all kinds, from microscopic to intergalactic scales.</td>
<td><a href="https://obofoundry.org/ontology/envo.html">ENVO</a></td>
</tr>
<tr>
<td>GO (Gene Ontology)</td>
<td>go</td>
<td>biological_process</td>
<td>2022/12/04</td>
<td>The Gene Ontology (GO) knowledgebase is the world's largest source of information on the functions of genes. This knowledge is both human-readable and machine-readable, and is a foundation for computational analysis of large-scale molecular biology and genetics experiments in biomedical research. Specific type: Biological process.</td>
<td><a href="http://geneontology.org/">Gene Ontology</a></td>
</tr>
<tr>
<td>GO (Gene Ontology)</td>
<td>go</td>
<td>cellular_component</td>
<td>2022/12/04</td>
<td>The Gene Ontology (GO) knowledgebase is the world's largest source of information on the functions of genes. This knowledge is both human-readable and machine-readable, and is a foundation for computational analysis of large-scale molecular biology and genetics experiments in biomedical research. Specific type: Cellular component.</td>
<td><a href="http://geneontology.org/">Gene Ontology</a></td>
</tr>
<tr>
<td>GO (Gene Ontology)</td>
<td>go</td>
<td>molecular_function</td>
<td>2022/12/04</td>
<td>The Gene Ontology (GO) knowledgebase is the world's largest source of information on the functions of genes. This knowledge is both human-readable and machine-readable, and is a foundation for computational analysis of large-scale molecular biology and genetics experiments in biomedical research. Specific type: Molecular function.</td>
<td><a href="http://geneontology.org/">Gene Ontology</a></td>
</tr>
<tr>
<td>ICD-O-3</td>
<td>icdo3</td>
<td>disease</td>
<td>unknown (2022)</td>
<td>The International Classification of Diseases Oncology (ICD-O) is a biomedical ontology for logical representation of the terms and relations related to the International Classification of Diseases (ICD).</td>
<td><a href="http://www.iacr.com.fr/index.php?option=com_content&amp;view=category&amp;layout=blog&amp;id=100&amp;Itemid=577">ICD-O-3</a></td>
</tr>
<tr>
<td>ICTV</td>
<td>ictv</td>
<td>virus</td>
<td>v3</td>
<td>The task of the International Committee on Taxonomy of Viruses (ICTV) is to develop a single, universal taxonomic scheme for all the viruses infecting animals (vertebrates, invertebrates and protozoa), plants (higher plants and algae), fungi, bacteria and archaea.</td>
<td><a href="https://ictv.global/taxonomy">ICTV</a></td>
</tr>
<tr>
<td>MDD</td>
<td>mdd</td>
<td>species</td>
<td>v1.7</td>
<td>Mammal species list with synonyms. An initial taxonomy of accepted names + synonyms based on v1.7 of the Mammal Diversity Database (MDD)</td>
<td><a href="https://www.mammaldiversity.org/index.html">MDD</a></td>
</tr>
<tr>
<td>MeSH</td>
<td>mesh</td>
<td>mesh</td>
<td>v2022</td>
<td>The Medical Subject Headings (MeSH), provided by the U.S. National Library of Medicine (NLM), is a controlled vocabulary thesaurus used for indexing articles for PubMed. In comparison with specialzed ontologies like the NCIt, MeSH is less granular and easily identified by Natural Language Processing thanks to synonyms.</td>
<td><a href="https://www.nlm.nih.gov/databases/download/mesh.html">MeSH</a></td>
</tr>
<tr>
<td>NCBI Taxonomy Clinic</td>
<td>ncbi</td>
<td>Species</td>
<td>2022/12/12</td>
<td>This terminology was created by the team and includes many species of clinical interest and related code from NCBI taxonomy collection.</td>
<td><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi">NCBI Taxonomy</a></td>
</tr>
<tr>
<td>NCBI Taxonomy Full</td>
<td>ncbi</td>
<td>Species</td>
<td>2022/12/12</td>
<td>The NCBI Taxonomy Database is a curated classification and nomenclature for all of the organisms in the public sequence databases.</td>
<td><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi">NCBI Taxonomy</a></td>
</tr>
<tr>
<td>NCI Thesaurus</td>
<td>ncit</td>
<td>disease</td>
<td>v22.11d</td>
<td>The NCI Thesaurus (NCIt) is used for disease mapping. It covers clinical care, translational and basic research, public information and administrative activities. Provided by the National Cancer Institute, this terminology is a standard for biomedical coding and reference, used both by public and private scientific partners worldwide. This NCI's reference terminology contains the NCI_CUI, the semantic type, a prefered term, some NCI and MeSH synonyms.</td>
<td><a href="https://ncithesaurus.nci.nih.gov/ncitbrowser/">NCIt</a></td>
</tr>
<tr>
<td>neXtProt</td>
<td>nextprot</td>
<td>gene</td>
<td>v2.26.0</td>
<td>Developed by the SIB (Swiss Institute of Bioinformatics) in 2008, the neXtProt human protein knowledgebase is a comprehensive human-centric discovery platform. More than 20,000 proteins were manually annotated and still updated. The provides to researchers a high-quality synonym for both protein and gene names.</td>
<td><a href="https://www.nextprot.org/">neXtProt</a></td>
</tr>
<tr>
<td>OTT (Open Tree of Life)</td>
<td>ott</td>
<td>species</td>
<td>v3.3</td>
<td>Open Tree of Life aims to construct a comprehensive, dynamic and digitally-available tree of life by synthesizing published phylogenetic trees along with taxonomic data. https://doi.org/10.5281/zenodo.3937750</td>
<td><a href="https://tree.opentreeoflife.org/about/taxonomy-version/ott3.3">OTT</a></td>
</tr>
<tr>
<td>PPI-PTM</td>
<td>ppiptm</td>
<td>ppi-ptm</td>
<td>unknown</td>
<td>xxx</td>
<td>[Manual]</td>
</tr>
<tr>
<td>ROBI (Relation Ontology - Biotic Interactions)</td>
<td>robi</td>
<td>biotic_interaction</td>
<td>unknown</td>
<td>RO is a collection of relations intended primarily for standardization across ontologies in the OBO Foundry and wider OBO library. It incorporates ROCore upper-level relations such as part of as well as biology-specific relationship types such as develops from. We extracted a subset of terms of interest for biotic interactions.</td>
<td><a href="https://obofoundry.org/ontology/ro.html">RO</a></td>
</tr>
<tr>
<td>UniProtKB/SwissProt</td>
<td>uniprot</td>
<td>gene</td>
<td>2022/04</td>
<td>The UniProt Knowledgebase (UniProtKB) is the central hub for the collection of functional information on proteins, with accurate, consistent and rich annotation. We used a large fraction of the reviewed subset UinProtKB/Swiss-Prot which includes records with information extracted from literature and curator-evaluated computational analysis.</td>
<td><a href="https://www.uniprot.org/downloads#uniprotkblink">UniProtKB/SwissProt</a></td>
</tr>
</tbody>
</table>



  



                
              </article>
            </div>
          
          
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
    <div class="md-copyright__highlight">
        Tagged entities are licensed under a <a href="http://creativecommons.org/licenses/by/3.0/ch/">Creative Commons Attribution 3.0 Switzerland License</a>.
    </div>
    <div>
        Questions specifically relating to these services may be sent to <a href="mailto:julien.gobeill@hesge.ch">julien.gobeill@hesge.ch</a>.
    </div>
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.tabs", "navigation.indexes", "content.code.copy"], "search": "../../assets/javascripts/workers/search.208ed371.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../../assets/javascripts/bundle.51198bba.min.js"></script>
      
    
  </body>
</html>